src/main/kotlin/org/valensas/simplyquartz/QuartzConfig.kt

```diff
@@ -0,0 +1,24 @@
+package org.valensas.simplyquartz
+
+import org.slf4j.LoggerFactory
+import org.springframework.boot.autoconfigure.quartz.QuartzDataSource
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.jdbc.DataSourceBuilder
+import org.springframework.context.annotation.Bean
+import org.springframework.stereotype.Component
+import javax.sql.DataSource
+
+
+@Component
```

Comment:
Burası bir kofigürasyon olduğu için `@Configuration` kullanılmalı, `@Component` değil